### ISSUE TITLE
What's the difference between in_img and gt_img?

### ISSUE THREAD

**Issue by alex04072000** [MenghanXia/InvertibleGrayscale#2]
Hi, thank you for sharing such a great work!
I want to ask that, since the network is trained in a self-supervised way. What is the meaning of gt_img in your data loader?
Is gt_img the same as in_img? (for the calculation of loss terms)
Thank you!

**Comment by MenghanXia** [comment]
Yes, the in_img and gt_img is the same. So, only one of them is needed.
